tests/modules/programs/opencode/my-theme.nix

```diff
@@ -0,0 +1,223 @@
+{
+  "$schema" = "https://opencode.ai/theme.json";
```

Comment:
Please add the schema dynamically to the configuration. See https://github.com/nix-community/home-manager/blob/722792af097dff5790f1a66d271a47759f477755/modules/programs/opencode.nix#L148-L151